Lib/pstats.py

```diff
@@ -432,28 +438,48 @@ def print_stats(self, *amount):
             print(file=self.stream)
         return self
 
-    def print_callees(self, *amount):
+    def print_callees(self, *amount, callees_sort_key=None, callees_filter=()):
         width, list = self.get_print_list(amount)
         if list:
             self.calc_callees()
 
+            sort_tuple = None
+            sort_type = None
+            if callees_sort_key:
+                if isinstance(callees_sort_key, str):
+                    callees_sort_key = (callees_sort_key,)
+                sort_tuple, sort_type = self.get_sort_tuple_and_type(*callees_sort_key)
+                print("   Callees ordered by: " + sort_type + "\n")
+            if not isinstance(callees_filter, tuple):
+                callees_filter = (callees_filter,)
             self.print_call_heading(width, "called...")
             for func in list:
                 if func in self.all_callees:
-                    self.print_call_line(width, func, self.all_callees[func])
+                    self.print_call_line(width, func, self.all_callees[func],
+                                         sort_tuple=sort_tuple, sort_type=sort_type, sel_list=callees_filter)
                 else:
                     self.print_call_line(width, func, {})
             print(file=self.stream)
             print(file=self.stream)
         return self
 
-    def print_callers(self, *amount):
+    def print_callers(self, *amount, callers_sort_key=None, callers_filter=()):
         width, list = self.get_print_list(amount)
         if list:
+            sort_tuple = None
+            sort_type = None
+            if callers_sort_key:
+                if isinstance(callers_sort_key, str):
+                    callers_sort_key = (callers_sort_key,)
+                sort_tuple, sort_type = self.get_sort_tuple_and_type(*callers_sort_key)
+                print("   Callers ordered by: " + sort_type + "\n")
```

Comment:
Does it have to be preceded by a space?
```
print(f"Callers ordered by: {sort_type}")
```
(print default line break)

Reply:
1. Yes, it matches indentation with the lines printed before it. See below.
2. I prefer f-strings but thought it would be better to follow existing style. Should I use f-strings anyway?
3. Existing output style puts a blank line before the call heading below, like:
    ```
       Ordered by: internal time
       List reduced from 965 to 1 due to restriction <'\\(has\\)'>

       Callers ordered by: internal time

    Function    was called by...
    [...]
    ```
    I can replace the `'\n'` with a `print()` if that is better?

    The blank line before "Callers" I would like to remove but that would require more mostly-unrelated changes.

Reply:
Any opinion on my changes to this?